src/anndata/experimental/merge.py

```diff
@@ -402,6 +417,24 @@ def _write_axis_annot(
     write_elem(output_group, axis_name, concat_annot)
 
 
+def _write_uns(groups, output_group, merge):
+    # function writes the .uns slot of merged AnnData objects to disk
+    # read and merge all .uns from input groups
+    uns = merge([read_elem(g["uns"]) for g in groups])
+    # normalizing the result if it is a single pd.Series
+    # making sure it is converted into a dataframe for consistency
+    if isinstance(uns, pd.Series):
+        uns = uns.to_frame()
+    # convert any nested series inside the merged dict to dataframes
+    # iterating over each element in dict and converting it to df
+    if isinstance(uns, dict):
+        for k, v in uns.items():
+            if isinstance(v, pd.Series):
+                uns[k] = v.to_frame()
+    write_elem(output_group, "uns", uns)
```

Comment:
I must be missing something here, but this would imply one could write out `uns` as a dataframe in some cases?